compiler/src/compiler/string_to_rcst.rs

```diff
@@ -614,27 +639,106 @@ mod parse {
         );
     }
 
+    #[instrument(level = "trace")]
+    fn text_interpolation(input: &str, curly_brace_count: usize) -> Option<(&str, Rcst)> {
+        let (mut input, mut opening_curly_braces) =
+            parse_multiple(input, opening_curly_brace, Some((curly_brace_count, true)))?;
+
+        if let Some((new_input, whitespace)) = single_line_whitespace(input) {
+            input = new_input;
+            let last = opening_curly_braces.pop().unwrap();
+            opening_curly_braces.push(last.wrap_in_whitespace(vec![whitespace]));
+        }
+
+        let (mut input, mut expression) = expression(input, 0, false, true, true).unwrap_or((
```

Comment:
If the expression wraps to a new line, it must be indented

```suggestion
        let (mut input, mut expression) = expression(input, indentation + 1, false, true, true).unwrap_or((
```

Reply:
Resolved by 511a54c